contracts/Core/storage/JobStorage.sol

```diff
@@ -5,6 +5,13 @@ import "../../lib/Structs.sol";
 
 
 contract JobStorage {
-    uint256 public numJobs;
     mapping (uint256 => Structs.Job) public jobs;
+    mapping (uint256 => Structs.Collection) public collections;
+
+    enum assetTypes { NONE, Job, Collection }
+
+    uint256[] public jobList;
+    uint256[] public collectionList;
+
```

Comment:
Hey I am not sure why do we need this both arrays - JobList and CollectionList. From code I think they are to store respective ids. why we need that but. Is there any reason. Becasue as of now As Per JobManager we are pushing values in respective arrays but I didnt see them getting used elsewhere. Can you please explain